src/server/server.js

```diff
@@ -79,12 +79,27 @@ app.post('/callback', (req, res) => {
     azureClient
         .callback(config.oidc.redirectUrl, params, { nonce })
         .then(tokenSet => {
-            res.cookie('speil', `${tokenSet['id_token']}`, {
-                secure: process.env.NODE_ENV !== 'development',
-                sameSite: true
-            });
-            req.session.spadeToken = tokenSet['access_token'];
-            res.redirect('/');
+            const accessToken = tokenSet['access_token'];
+            const requiredGroup = config.requiredGroup;
+            if (
+                !accessToken ||
+                !authsupport.isMemberOf(requiredGroup, accessToken)
+            ) {
+                console.log(
+                    `'${authsupport.nameFrom(
+                        accessToken
```

Comment:
`accessToken` kan kanskje være `undefined` her, ref if-sjekken over?

Reply:
Sant, lyt fikse

Reply:
authsupport.nameFrom tar høyde for det